netty/src/main/java/io/atomix/catalyst/transport/NettyTls.java

```diff
@@ -15,71 +15,62 @@
  */
 package io.atomix.catalyst.transport;
 
-import javax.net.ssl.KeyManager;
-import javax.net.ssl.KeyManagerFactory;
-import javax.net.ssl.SSLContext;
-import javax.net.ssl.SSLEngine;
-import javax.net.ssl.SSLSocket;
-import javax.net.ssl.SSLSocketFactory;
-import javax.net.ssl.TrustManager;
-import javax.net.ssl.TrustManagerFactory;
 import io.atomix.catalyst.util.Assert;
-import java.io.FileInputStream;
-import java.security.KeyStore;
-import java.io.File;
-
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import javax.net.ssl.*;
+import java.io.File;
+import java.io.FileInputStream;
+import java.security.KeyStore;
+
 /**
  * Netty TLS.
  *
  * @author <a href="http://github.com/electrical">Richard Pijnenburg</a>
  */
-
-public class NettyTls {
-
+final class NettyTls {
   private static final Logger LOGGER = LoggerFactory.getLogger(NettyTls.class);
-  private static boolean ts_use_ks = true;
```

Comment:
These fields did not need to be class/object fields and seem to be fine as local variables in the `initSslEngine` method.